envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua

```diff
@@ -8,21 +8,26 @@ function envoy_on_request(handle)
     local request_id = first_header_value_from_list(request_id_header_names, handle)
     local trusted_header_name = handle:metadata():get("trusted_client_identity_header") or ""
     local client_name = ""
+    local trusted_client = "false"
     if trusted_header_name ~= "" then
         client_name = handle:headers():get(trusted_header_name) or ""
+        if client_name ~= "" and handle:connection():ssl() ~= nil then
```

Comment:
`and handle:connection():ssl() ~= nil` is not needed - if trusted header is set, it is trusted, period.

Reply:
fixed